server/routes/client.js

```diff
@@ -2,7 +2,12 @@ const router = require("express").Router();
 const AuthController = require("../controllers/AuthController");
 const { check, validationResult, body } = require("express-validator");
 const ClientDashboardController = require("../controllers/ClientDashboardController");
-const { Gig, UserClient, UserPerformer } = require("../database");
+const {
+  Gig,
+  UserClient,
+  UserPerformer,
+  getPerformerDetails,
```

Comment:
shouldn't be importing anything else from database, getPerformerDetails should be removed, pretty sure its apart of your controller so it probably doesn't exist in the data database file